client/src/store.ts

```diff
@@ -0,0 +1,16 @@
+import { createStore, applyMiddleware } from "redux";
+import { composeWithDevTools } from "redux-devtools-extension";
+import thunk from "redux-thunk";
+import rootReducer from "./reducers";
+
+const initialState = {};
```

Comment:
Create interface for you root state and describe it

Reply:
done

Reply:
done